src/main/java/com/newbarams/ajaja/module/plan/presentation/PlanController.java

```diff
@@ -71,13 +72,16 @@ private Long parseUserId(String accessToken) {
 
 	@PostMapping
 	@ResponseStatus(CREATED)
-	public AjajaResponse<PlanResponse.Create> createPlan(
+	public AjajaResponse<Void> createPlan(
 		@UserId Long userId,
 		@RequestBody PlanRequest.Create request,
-		@RequestHeader(name = "Month") @Min(1) @Max(12) int month
+		@RequestHeader(name = "Month") @Min(1) @Max(12) int month,
+		HttpServletResponse response
 	) {
-		PlanResponse.Create response = createPlanService.create(userId, request, month);
-		return AjajaResponse.ok(response);
+		Long planId = createPlanService.create(userId, request, month);
+		response.addHeader("Location", "plans/" + planId);
+
+		return AjajaResponse.ok();
```

Comment:
이럴 때 이제 `ResponseEntity`를 사용하면 좋습니다!
예를 들면 이렇게 말이죠!
```suggestion
		Long planId = createPlanService.create(userId, request, month);
		URI uri = URI.create("plans/" + planId);
		return ResponseEntity.created(uri).body(AjajaResponse.ok());
```

Reply:
오 훨씬 보기좋네요ㅎㅎ 수정했습니다!